src/Kafka/KafkaInstances/components/InstancesTable.tsx

```diff
@@ -184,8 +186,10 @@ export const InstancesTable = <T extends KafkaInstance>({
         );
       }}
       renderActions={({ row, ActionsColumn }) => {
-        const changeOwnerEnabled = canChangeOwner(row);
+        const changeOwnerEnabled =
+          !canChangeOwner(row) || instanceSuspended(row);
```

Comment:
Using the above suggestion, there is no need to change this. The logic should all live in `KafkaInstances`.